src/model/transaction/Transaction.ts

```diff
@@ -233,6 +229,34 @@ export abstract class Transaction {
         );
     }
 
+    /**
+     * Signs raw transaction with the given private key
+     * @param {string} privateKey - Private key of the signer account
+     * @param {number[]} rawTxSigningBytes - Raw transaction siging bytes
+     * @returns {number[]} Signature byte array
+     */
+    public static signRawTx(privateKey: string, rawTxSigningBytes: number[]): number[] {
```

Comment:
Should the public method use Uint8Array for input and output rather than number[]?

Reply:
Makes sense, updated.